redhat-developer/developers.redhat.com
# [SURGE 3060] Product Try-It Hero

### JIRA Issue Link
* https://github.com/redhat-developer/developers.redhat.com/issues/3060

### Verification Process
1. View Product Try-It Hero: `/products/codeready-workspaces`
2. Check that it matches FE: https://redhat-developer.github.io/rhd-frontend/components/product-try-it/
